pkg/river/token/builder/builder.go

```diff
@@ -166,9 +171,21 @@ func (b *Body) encodeFields(rv reflect.Value) {
 	}
 
 	fields := rivertags.Get(rv.Type())
+	clone := reflect.New(rv.Type()).Elem()
```

Comment:
Rather than calling it `clone`, what do you think about a name more descriptive of what we're using it for, like `defaults`?